functionalTest/test/utilities/reusableLibrary.js

```diff
@@ -1,6 +1,21 @@
-/**
- * Created this file to include all reusable methodes in future.
- */
-class reusableLibrary {}
+import dashboard from "../pageobjects/dashboardPage";
 
+class reusableLibrary {
+  sliderTrackResult() {
+    dashboard.selectTimeSpan("select-all").click();
+    expect(dashboard.sliderTrack).toBeDisplayed();
```

Comment:
What are the results of clicking the buttons now? ie what changes between button clicks?